tools/cldr-code/src/main/resources/org/unicode/cldr/util/downgrade.txt

```diff
@@ -0,0 +1,11 @@
+# Format is 
+#    locale-regex ; path-regex ; value-regex
+# Where a field is missing, it matches all values
```

Comment:
some comments, not blocking

- `.*` might be less err prone than missing

Reply:
I was debating whether to make it be a match or not. We already do this processing in RegexLookup, so makes sense to centralize